mlrun/common/schemas/alert.py

```diff
@@ -101,6 +101,17 @@ class ResetPolicy(StrEnum):
     AUTO = "auto"
 
 
+class AlertNotification(pydantic.BaseModel):
+    notification: Notification
+    cooldown_period: Annotated[
+        str,
```

Comment:
What would be the format of the cooldown period string? The description here doesn't hint on it, and you don't do any validation anywhere on the format. 
If you keep this as string (for example, since date-time objects cannot serialize very well), then I would expect at least some validation and parsing of these strings, and also a documentation about it.